### PR TITLE
chore: publish package with updates

### PR DESCRIPTION
**Pull request by TerryMooreII**
Summary:
The nprogress library was several years out of date.  We had already
forked this to to use a fix that was on master but never released.
This PR also updates all devDependancies and fixes the tests to work
with the latest version of mocha, chai, and jsdom.

BREAKING CHANGE: forked the project and republish under the @logdna namespace

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,23 +1,26 @@
 {
-  "name": "nprogress",
+  "name": "@logdna/nprogress",
```
Comment (esatterwhite) [inline package.json:2]: you'll need to add a `publishConfig` key to set the access for the scope to `public`
```json

{
  "private": false
,  "publishConfig": {
      "access": "public"
    }
}

**Review by esatterwhite — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (TerryMooreII) [reply package.json:2]: done

**Review by esatterwhite — APPROVED** [review]

**Review by esatterwhite — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,23 +1,30 @@
...
+  },
   "repository": {
     "type": "git",
-    "url": "https://github.com/rstacruz/nprogress.git"
+    "url": "https://github.com/logdna/nprogress.git"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "nyc mocha",
+    "test:ci":  "mocha test --reporter mocha-junit-reporter --reporter-options mochaFile=./coverage/test.xml"
```
Comment (esatterwhite) [inline package.json]: you need to add the `release` script

File: package.json [hunk 3]
```diff
@@ -1,23 +1,30 @@
...
-    "jsdom": "^5.4.3",
-    "mocha": "^2.2.4",
-    "mocha-jsdom": "^0.3.0"
+    "chai": "^4.3.4",
+    "jquery": "^3.6.0",
+    "jsdom": "^16.6.0",
+    "mocha": "^8.4.0",
+    "mocha-jsdom": "^2.0.0",
+    "mocha-junit-reporter": "^2.0.0",
+    "nyc": "^15.1.0"
```
Comment (esatterwhite) [inline package.json]: you need to add the semantic release package too

Comment (esatterwhite) [reply package.json]: https://github.com/logdna/stdlib-node/blob/main/package.json#L60-L61

File: package.json (hunk 2 above)
Comment (TerryMooreII) [reply package.json]: yep, I just added that

**Review by esatterwhite — APPROVED** [review]

**Comment by mdeltito** [comment]
likely doesn't matter, but if we want to publish this as a `1.0.0` (rather than continue the versioning where the old package left off) you can mark it as a major/breaking in the commit type. Something like:

```
chore!: publish package with updates

BREAKING CHANGE: forked the project and republish under the @logdna namespace
...
```

**Review by mdeltito — APPROVED** [review]

**Pull request merged**